scenarios/README.md

```diff
@@ -0,0 +1,50 @@
+# Scenarios
+
+This folder contains various projects conducted in cyclus.
+
+## 2014-E&S
+Summary: Transition scenarios for evaluation groups from 
+the 2014 E&S Study from the U.S. Department of Energy.
+
+
+## 2017-bae-europe
+Summary: French fleet simulation with random lifetime 
+extension and sensitivity analysis.
+
+
+## aqueous_reprocess
+Summary: A simulation where uox_waste is output as 
+reprocess_waste, uox_U and uox_Pu to get familiar 
+with the separations cycamore archetype. It 
+contains a source, separations and sink facility.
+
+
+## eu_1950-2020
+Summary: A project simulating the number of reactors 
+and net capacity of a hypothetical system.
+
+
+## EU_1950_to_2020_reprocess
+Summary: A project building off of eu_1950_2020 to
+simulate the number of reactors and net capacity of 
+a hypothetical system with reprocessing capabilities.
+
+
+## haleu
```

Comment:
Might be a good idea to link to my thesis once it's published.

Reply:
We'll make an issue for that!